lib/wings/valkyrie/persister.rb

```diff
@@ -23,15 +23,16 @@ def initialize(adapter:)
       # Persists a resource using ActiveFedora
       # @param [Valkyrie::Resource] resource
       # @return [Valkyrie::Resource] the persisted/updated resource
-      def save(resource:)
+      def save(resource:, valid: true)
```

Comment:
by default we will not validate the converted `af_object` because validations should have been run on the Valkyrie created object. 

however, there are situations where we will have created an ActiveFedora object while `Hyrax.config.use_valkyrie?` is TRUE. (see: spec/controllers/hyrax/generic_works_controller_json_spec.rb:59)

Reply:
Hmm…thinking about this parameter name `valid`?  It is a bit confusing.

Does it mean "Treat this record as valid?"  Or does it mean "validate this record".  Looking at how it is later used, it looks like it could be `perform_af_validation: false`